client/src/components/dashboard/deploy/DeployPage.js

```diff
@@ -8,6 +8,7 @@ import { Button, Spinner } from 'react-bootstrap';
 const DeployPage = ({ teamId }) => {
   const dispatch = useDispatch();
   const loading = useSelector((state) => state.deploy.loading);
+  // TODO: refactor this into useState
```

Comment:
The refactoring in this file will happen in another ticket as this PR is already too big